### ISSUE TITLE
Cannot load local files with non-ascii characters in path

### ISSUE THREAD

**Issue by vanmonday01** [silveredgold/beta-censoring#13]
Seems like the backend does not resolve percent encoding correctly.

Stacktrace:
```
BP - Error response from censoring request! Error encountered while censoring this image: System.IO.FileNotFoundException: Could not find file 'C:\Users\ms\Downloads\%C3%9F.jpg'.
File name: 'C:\Users\ms\Downloads\%C3%9F.jpg'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String , FileMode , FileAccess , FileShare , FileOptions )
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String , FileMode , FileAccess , FileShare , FileOptions , Int64 )
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String , FileMode , FileAccess , FileShare , FileOptions , Int64 )
   at System.IO.FileStream..ctor(String , FileMode , FileAccess , FileShare , Int32 , FileOptions )
   at System.IO.File.ReadAllBytesAsync(String , CancellationToken )
   at CensorCore.ImageSharpHandler.LoadImage(String path) in /home/runner/work/censor-core/censor-core/src/CensorCore/ImageSharpHandler.cs:line 57
   at CensorCore.AIService.RunModel(String url, MatchOptions options) in /home/runner/work/censor-core/censor-core/src/CensorCore/AIService.cs:line 124
   at BetaCensor.Core.Messaging.CensorImageRequestHandler.RunModel(CensorImageRequest request, MatchOptions options) in /home/runner/work/beta-censoring/beta-censoring/src/BetaCensor.Core/Messaging/CensorImageRequestHandler.cs:line 71
   at BetaCensor.Core.Messaging.CensorImageRequestHandler.Handle(CensorImageRequest request, CancellationToken cancellationToken) in /home/runner/work/beta-censoring/beta-censoring/src/BetaCensor.Core/Messaging/CensorImageRequestHandler.cs:line 54 (Could not find file 'C:\Users\ms\Downloads\%C3%9F.jpg'.) a169abde-9789-4ab7-b944-1c293d6d79e0
```

Using Beta Censoring 0.0.9, Beta Protection 0.1.7. Windows 10 64x.

**Comment by silveredgold** [comment]
Oh interesting, I hadn't tested that. Let me look into it!

**Comment by silveredgold** [comment]
This should be fixed in v0.0.11
